x-pack/test/functional/services/ml/job_wizard_common.ts

```diff
@@ -619,5 +619,67 @@ export function MachineLearningJobWizardCommonProvider(
         await testSubjects.existOrFail(expectedSelector);
       });
     },
+
+    async getAnnotationSwitchCheckedState(): Promise<boolean> {
+      const subj = 'mlJobWizardSwitchAnnotations';
+      const isSelected = await testSubjects.getAttribute(subj, 'aria-checked');
+      return isSelected === 'true';
+    },
+
+    async getAnnotationsSwitchCheckedState(expectedValue: boolean) {
+      const actualCheckedState = await this.getAnnotationSwitchCheckedState();
+      expect(actualCheckedState).to.eql(
+        expectedValue,
+        `Expected annotations switch to be '${expectedValue ? 'enabled' : 'disabled'}' (got '${
+          actualCheckedState ? 'enabled' : 'disabled'
+        }')`
+      );
+    },
+
+    async toggleAnnotationSwitch(toggle: boolean) {
+      const subj = 'mlJobWizardSwitchAnnotations';
+      if ((await this.getAnnotationSwitchCheckedState()) !== toggle) {
+        await retry.tryForTime(5 * 1000, async () => {
+          await testSubjects.clickWhenNotDisabledWithoutRetry(subj);
+          await this.getAnnotationSwitchCheckedState();
+        });
+      }
+    },
```

Comment:
We have a similar code snippet copy-pasted in so many places. Can you please create reusable methods in `x-pack/test/functional/services/ml/common_ui.ts`? 
- getSwitchChechedState(testSubj: string)
- toggleSwitchIfNeeded(testSubj: string, targetState: boolean)

Reply:
https://github.com/wayneseymour/kibana/commit/6602b290f5763c857fa68cde24d1710191022648